Configuration/Geometry/README.md

```diff
@@ -37,5 +39,7 @@ Several detector combinations have been generated:
 * D44 = T14+C6+M3+I7+O2+F3
 * D45 = T15+C8+M3+I10+O3+F2
 * D46 = T15+C9+M3+I10+O3+F2
+* D47 = T15+C10+M3+I10+O3+F3
+* D48 = T16+C10+M3+I10+O3+F3
```

Comment:
I don't think a tracker test workflow needs to include the HF nose. I would make either D45 or D46 the basis for D48.

Reply:
Are C10 and F3 not the latest descriptions available?
In that case, why not having the latest Tracker description with them, and compare D48 versus D47?
It would avoid having latest subdetectorA with old subdetectorB and latest subdetectorB with old subdetectorA. Which would imply that when one wants to access the latest CMS geometry, it does not exist?

Reply:
C10 and F3 are the latest version of the HF nose. This is a detector R&D effort that is not yet an approved part of the Phase 2 upgrade project. Combining tracker tests with this will just lead to confusion and inconsistencies.

Reply:
Ha I didn't know C10 and F3 were purely in an hypothetical phase for now.
Ok I will simply revert my last commit then.